xml/chapter1/section2/subsection3.xml

```diff
@@ -49,7 +49,11 @@
     <LATEXINLINE>$k_2$</LATEXINLINE> independent of
     <LATEXINLINE>$n$</LATEXINLINE> such that
     <LATEX>
-      \[k_1f(n) \leq R(n) \leq k_2f(n)\]
+      \[
+      \begin{array}{lll}
+      k_1f(n) &amp; \leq &amp; R(n) \quad \leq \quad k_2f(n)
```

Comment:
Very minor comment: the subscripts of k overlap with the serif of the f. Moving from k_1 and k_2 to e.g. k and k' would solve this.

Reply:
Not proposing that we make that change. Just pointing it out.

Reply:
I've added thin spaces \, after k_1 and k_2 to avoid the overlap. This now looks the same as in SICP.